vim-patch:8.1.{205,206,208,506,546,547,751}

janlazo opened the pull request:
These should have been merged before merging 8.1.0544.

janlazo committed:
vim-patch:8.1.0206: duplicate test function name

Problem:    Duplicate test function name.
Solution:   Rename both functions.
https://github.com/vim/vim/commit/cd96eef3a869557bd3d2d4497861d87cb525db06

janlazo committed:
vim-patch:8.1.0205: invalid memory access with invalid modeline

Problem:    Invalid memory access with invalid modeline.
Solution:   Pass pointer limit. Add a test. (closes vim/vim#3241)
https://github.com/vim/vim/commit/9cf4b5005f12ce1d6692266140bdda05d0312d79

janlazo committed:
vim-patch:8.1.0506: modeline test fails when run by root

Problem:    Modeline test fails when run by root.
Solution:   Set 'modeline' for the test. (James McCoy, closes vim/vim#3592)
https://github.com/vim/vim/commit/9691f82f862ed18c2e3e48f9d2cd902bb947f803

N/A patches:

vim-patch:8.1.0208: file left behind after running individual test

janlazo commented on src/nvim/option.c:
```diff
@@ -4957,19 +4957,20 @@ char *set_option_value(const char *const name, const long number,
-  int key;
+  int key = 0;
   int modifiers;
+  const char_u *arg = arg_arg;
```
This was in 8.1.0205. Seems unnecessary when `find_special_key` accepts `const char_u *`.

justinmk replied:
Seems unusual, but I guess it's fine.

justinmk commented:
> These should have been merged before merging 8.1.0544.

Thanks for fixing this!

janlazo committed:
vim-patch:8.1.0546: modeline test with keymap fails

Problem:    Modeline test with keymap fails.
Solution:   Check that the keymap feature is available.
https://github.com/vim/vim/commit/4ace6ab7e72151d78f1563b23b498ed20fa00211

janlazo committed:
vim-patch:8.1.0547: modeline test with keymap still fails

Problem:    Modeline test with keymap still fails.
Solution:   Check that the keymap feature is available for the failure assert.
https://github.com/vim/vim/commit/3067a4dd0d768d1e4a47cf3c100f3e2b462717d7

janlazo committed:
vim-patch:8.1.0751: some regexp errors are not tested

Problem:    Some regexp errors are not tested.
Solution:   Add a test function.
https://github.com/vim/vim/commit/6057ed4720eaa1d546bbd234e6957ec0163e0e9b

janlazo committed:
test/old: remove duplicates, run test_tabpage.vim
